model/project.go

```diff
@@ -431,7 +431,7 @@ func (tt TaskIdTable) GetIdsForAllTasks(currentVariant, taskName string) []strin
 }
 
 // TaskIdTable builds a TaskIdTable for the given version and project
-func NewTaskIdTable(p *Project, v *version.Version) TaskIdConfig {
+func NewTaskIdTable(p *Project, v *version.Version, sourceRev, defID string) TaskIdConfig {
```

Comment:
I worry a bit about these two arguments and wanting to have an args struct for this too, but I think it's fine.

Reply:
I think it's worth adding an args struct if we add more to this

Reply:
what's the limit? why not now?

Reply:
My limit was "when it feels unwieldy" but it doesn't feel unwieldy to me yet